client/router.js

```diff
@@ -173,29 +173,25 @@ Router.prototype.middleware = function(middlewareFn) {
 
 Router.prototype.ready = function() {
   var currentRoute = this.current().route;
+  var globalRoute = this._globalRoute;
+
   if(!currentRoute) {
     return false;
   }
 
   var subscriptions;
   if(arguments.length === 0) {
-    subscriptions = _.values(currentRoute.getAllSubscriptions());
+    subscriptions = _.values(globalRoute.getAllSubscriptions());
+    subscriptions.concat(_.values(currentRoute.getAllSubscriptions()));
   } else {
     subscriptions = _.map(arguments, function(subName) {
-      return currentRoute.getSubscription(subName);
+      return globalRoute.getSubscription(subName) || currentRoute.getSubscription(subName);
```

Comment:
I think can check this is a better way. Like this:

~~~js
function checkSubs(subs) {
  return _.every(subs, function(sub) {
    return sub.ready();
  });
}

var isReady = 
  checkSubs(currentRoute.getAllSubscriptions) &&
  checkSubs(globalRoute.getAllSubscriptions);

return isReady;
~~~

I think that's simple.
What do you think?